libcudacxx/.upstream-tests/test/std/utilities/memory/specialized.algorithms/specialized.addressof/addressof.pass.cpp

```diff
@@ -16,6 +16,10 @@
 
 #include "test_macros.h"
 
+#ifdef TEST_COMPILER_CLANG_CUDA
+#include <new>
+#endif // TEST_COMPILER_CLANG_CUDA
```

Comment:
Note, we need to address this properly 

Currently we cannot use `<new>` internally but I have some preliminary patches to make it work